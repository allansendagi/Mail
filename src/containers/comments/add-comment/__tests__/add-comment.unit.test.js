import React from 'react';
import {shallow, configure} from 'enzyme';
import {AddComment} from '../add-comment.container';
import Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter()});
describe('AddComment', () => {
  test('AddComment renders correctly', () => {
    const wrapper = shallow(
      <AddComment/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});