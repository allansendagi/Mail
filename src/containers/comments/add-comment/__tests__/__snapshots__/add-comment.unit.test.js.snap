// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddComment AddComment renders correctly 1`] = `ShallowWrapper {}`;
